lib/katello/engine.rb

```diff
@@ -41,7 +41,6 @@ class Engine < ::Rails::Engine
           :upload_chunk_size => 1_048_575, # upload size in bytes to pulp. see SSLRenegBufferSize in apache
           :sync_threads => 4,
           :sync_KBlimit => nil,
```

Comment:
Remove the trailing comma to move past the build error
```suggestion
          :sync_KBlimit => nil
```

Reply:
IMHO trailing commas should be mandatory but for now I applied this.